src/Akeneo/Connectivity/Connection/back/Infrastructure/Marketplace/ContentSecurityPolicy/MarketplaceContentSecurityPolicy.php

```diff
@@ -15,7 +17,10 @@ final class MarketplaceContentSecurityPolicy implements ContentSecurityPolicyPro
     public function getContentSecurityPolicy(): array
     {
         return [
-            'img-src' => ["marketplace.akeneo.com"],
+            'img-src' => [
+                '*.apps.akeneo.com',
```

Comment:
```suggestion
                'apps.akeneo.com',
```

We are displaying images from apps subdomains, we are displaying images from `app.akeneo.com` which is our App Store.